graphics/wxWidgets-3.2/Portfile

```diff
@@ -132,4 +127,19 @@ variant stdlib description {add support for various standard library features} {
                             --enable-std_string_conv_in_wxstring
 }
 
+# handle C++11 support
+if {${configure.cxx_stdlib} eq "libstdc++"} {
```

Comment:
Not sure I follow the logic here. Why not just make c++11 unconditional for both runtimes. I don’t get why its only a variant when the runtime is libc++ ?

Reply:
Trial and error to figure out what works and what doesn't, and the stdlib was the common factor (not the OS):

(1) wxWidgets-3.2 does not _require_ c++11 to build, hence c++11 should be an option where it makes sense as forcing c++11 just constrains / reduces compiler options (albeit minimally these days given how old c++11 is & how compatible compilers are and how relatively simple it is to install a compatible compiler via MacPorts on pretty much any OSX from 10.4 to 10.14). In my testing, if stdlib is libc++ then it doesn't matter if this option is enabled or not from a compatibility perspective with other ports that depend on this port's ABI. That said, the option works and maybe would be useful sometime 

(2) When using stdlib of libstdc++ and without the c++11 option, the resulting binaries don't work with binaries from other ports that do require c++11 (and thus use macports-libstdc++). I suppose I could just switch the stdlib to "macports-libstdc++", but I doubt that's very robust against older Xcode compilers; moving to c++11 and switching the stdlib virtually guarantees that the build will work and be compatible with other libstdc++ installs. Hence forcing c++11 in the case of stdlib of libstdc++.

Reply:
Ok, thanks for the more detailed explanation.

I still think I would personally favour simplifying the portfile and just enabling c++11 across the board. I don’t really buy into the argument that the reduction in compatible compilers that requiring the cxx11 portgroup results in is a problem. For most oses, the more up to date ones, it does nothing as the system clang compilers are already c++11 compatible. For those that are not, it just means they will use macports clang 5.0, which anyway this older systems will be using for more and more ports. So, honestly, I personally don't really see the argument against just enabling it across the board...